include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_for.h

```diff
@@ -67,10 +90,11 @@ struct __parallel_for_small_submitter<__internal::__optional_kernel_name<_Name..
             oneapi::dpl::__ranges::__require_access(__cgh, __rngs...);
 
             __cgh.parallel_for<_Name...>(sycl::range</*dim=*/1>(__count), [=](sycl::item</*dim=*/1> __item_id) {
+                // Disable vectorization and multiple iterations per item within the brick to evenly spread work across
+                // compute units.
+                __pfor_params<false, false, _Ranges...> __params;
```

Comment:
```suggestion
                __pfor_params<false /*__can_vectorize_brick*/, false /*__can_brick_process_multiple_iters*/, _Ranges...> __params;
```
Maybe this will make it a little clearer.

Reply:
Done here and in 2136.